__init__.py

```diff
@@ -134,9 +124,11 @@ def update_weather(self):
             self._load_skill_apis()
         if self.weather_api:
             current_weather_report = self.weather_api.get_current_weather_homescreen()
+            self.gui["weather_api_enabled"] = True
```

Comment:
Should this be conditional on `weather_code` and/or `weather_temp` being non-null? Since weather_api is configurable, it's possible some implementation will reply with empty data if a backend is down, etc.

Reply:
The issue with this is that weather_code / weather_temp is not available and not even set as a key before the any weather API loads so it cannot be used to determine if the api is functional, this is simply dependent on the weather api loading at all, if the weather api fails and sends null values that should probably be handled differently in the qml itself which i will add on another PR

Reply:
Sounds good to handle separately